### ISSUE TITLE
Let atom detach Asciidoc Preview

### ISSUE THREAD

**Issue by manueljordan** [atom/atom#21993]
For Atom with the `Asciidoc Preview` plugin installed, would you consider add the detach support?
It mostly if we have two monitors.

If this feature can be enable in general, would be great!

**Comment by ThatXliner** [comment]
Do you mean like make the pane into another window? I remember that you *can* do that...

**Comment by manueljordan** [comment]
Yes, detach the preview tab from the main Window and get it how a secondary Window.

If is possible please let me know.

**Comment by ThatXliner** [comment]
(Note to maintainers: don't close this because "atom treats issues as a backlog" yet. If what I'm thinking isn't what @manueljordan wants, then it's an enhancement)

**Comment by manueljordan** [comment]
In the main window having some `.adoc` files opened, I am able to write/edit them. with the `Asciidoc Preview` opened, _by default_ the main window is split in two parts in vertical, left part the source code and in the right the `Asciidoc Preview`. Each one in its own tab. All the tabs are in the same and unique main window. I want detach from the right part the tab that represents the `Asciidoc Preview` into a new window to move it into a secondary monitor. And BTW keeping that if I edit the source code, the `Asciidoc Preview` be still able to reflect the new changes.

Other scenario - I can have many `.adoc` files opened, lets say 4, and 4 `Asciidoc Preview` for them, for any of these 4, I want detach it

I hope it be more clear now.

Thanks for your understanding.
